### ISSUE TITLE
Use this Config and dont worry about ban

### ISSUE THREAD

**Issue by ramsamdam** [finndev/PokeBuddy#1058]
1. Fix a Location for Lvl up, like i have reached from lvl 1 to lvl 34 now without going anywhere except Pier 39, i know this will not help you build a fancy pokedex but still

2. Do not copy exact coordinates as you get of Nests/from site show some randomness

3. Use a VPN , like i am using an IP of San Francisco for botting in Pier 39
VPN like this https://www.privatetunnel.com/home/?referral=04CK5TENYN

4. Walking speed i have used less than 12 always

5. I keep restarting bot and changing IP after every 1-2 Hours

6. I dont keep the tick on Evolve / Transfer Bad / Recycle all the time the bot is on, after every 15-20 mins tick on one and then remove it

**Comment by MariaBitterlich** [comment]
Do you have incubators off as well?

> I dont keep the tick on Evolve / Transfer Bad / Recycle all the time the bot is on, after every 15-20 mins tick on one and then remove it

Do you know a reason for it or just a guess to be safe?

**Comment by ramsamdam** [comment]
Yes Incubators also off, 
I am just trying to make the bot mimic the human gameplay as close as possible, because they wont be checking each account individually unless you are reported

**Comment by ramsamdam** [comment]
One more Very important thing after bot running for 1-2 Hours your player wont be at the initial location where you had started the bot, if you want to start the bot immediately do the following steps,

1. Note down the current latitude and longitude, dont go more than a precision of 2 decimal places
2. After that click on stop, wait till it actually stops
3. Refresh VPN to get new IP
4. Enter the latitude longitude you noted down in the settings
5. Now click on Start

This will prevent an immediate soft ban

**Comment by MariaBitterlich** [comment]
@ramsamdam, yes, after they are going after bots now the race is on for the most humanlike bot. This is the only way to go on with botting. Surely, with less XP / hour than in the golden days, but still...

I want to understand, why those options are not human-like? Can you explain, why botting without incubators should be more human-like?

Furthermore, why are the other options not human-like?

**Comment by ramsamdam** [comment]
What i can think according to my domain knowledge is they have setup some algorithm to detect botting and then ban users accordingly. So i try to make my gameplay as unique as possible why i am doing that ?

You can test this by making 2 Accounts run them simultaneously on the bot and see, they will be following the same path, catching the same pokemons and gaining almost the same exp/hr which becomes quite obvious that they are on bot, this will lead to IP Ban.

Other than this about the remaining options,

If i play PoGo normally i wont be simultaneously walking from one pokestop to another which a bot does in 4-6 secs, and discarding all the items in my bags , catching all the pokemons with bonus exp, evolving pokemons as soon as i get the required candy etc. all is done within 4-6 secs which is not possible when you actually play PoGo.

Now about the incubator thing, 
When you play normally you know that when the egg hatches you get a pop up and you click on the egg and it displays the pokemon, i just have this feeling that this pop might be unhandled by the bot at the present scenario so i avoid using it.

Their Algorithm just needs 1 situation/move to flag us and then ban us, so we need brainstorm for ways in which they can detect these bots and try avoiding them.

**Comment by MariaBitterlich** [comment]
> Their Algorithm just needs 1 situation/move to flag us and then ban us, so we need brainstorm for ways in which they can detect these bots and try avoiding them.

Couldn't agree more. The bot would just need to randomize the perfectness for all of it's actions and alter the walking pattern with bezier curves etc.

First I was afraid that the pokebuddy devs throwing the towel in because of the legal pressure from Niantec, but recently they released a new version. Do you know what exactly has been changed?

**Comment by karimavine** [comment]
And for how long should i bot/stop @ramsamdam

**Comment by ramsamdam** [comment]
@MariaBitterlich have no idea about the changes they have made but definitely there is nothing generating randomness as i had 2 accounts on bot simultaneously starting from the same location, they were following the exact same path and catching almost 19/20 same pokemons

There is a tradeoff between xp/hour and randomness with the condition to avoid ban lets see if we can reach 40 before that

@karimavine that is up to you but i usually stop it after an interval of 2 hours or so never had it running overnight

**Comment by karimavine** [comment]
@ @ramsamdam  and how long is the break ?

**Comment by MariaBitterlich** [comment]
2 hours was the longest time the bot ran on my side, after that it was stuck or IP banned. My bot runs in my office, beause at home a hava a fix IP - so I let the bot run into the IP ban and if I'm back I reset IP and start again. Of course, that way I risk a permaent ban, but I think perm bans are more after very high levels - I'm 21...

**Comment by ramsamdam** [comment]
@karimavine 10 Minutes is the lower limit  , you can start immediately also i have done that quite a few times but 10 minutes wont be of any harm, and make sure you start from the coordinates where you had left on

@MariaBitterlich Use VPN at your home, i had reached Lvl21 in a day as far as i remember

**Comment by klykly1** [comment]
@ramsamdam sir it will not get ban? do you still use poke buddy? i have a problema bout poke buddy can you help me thanks :)

**Comment by psblackz** [comment]
use auto mouse clicker and VM VirtualBox, and why? use VirtualBox to isntall another windows to dont stop what u doing in your pc with Auto Mouse clicker... auto mouse clicker for 10 in 10 min stop bot and restart again

**Comment by psblackz** [comment]
and we dont to stop by urself.. just leave like a bot in old days

**Comment by ramsamdam** [comment]
@klykly1 I have not experienced a ban up till now, so just follow the steps stated above and hope for the best

**Comment by karimavine** [comment]
@ramsamdam 1 last thing :D  do i have to use vpn if im farming in my town ?

**Comment by klykly1** [comment]
@ramsamdam do you have some skype sir ram? so i can easily ask a question if tis okay to have your skype?

**Comment by ramsamdam** [comment]
@karimavine Yes because IP Ban may not be solely on the basis of location, it can be done if they detect that they are getting large number of requests/updates from a particular ip thus conforming the bot behavior, leading to IP Ban

See as i said they just need our one mistake to ban us, try not to make one.

**Comment by Xfro455** [comment]
privatetunnel connecting not working :(

**Comment by MariaBitterlich** [comment]
@ramsamdam, botting with [VPN ](https://www.privatetunnel.com/home/?referral=BWPNR9F03E)and it works just fine. Thanks!
